src/Tomcat/TomcatController.ts

```diff
@@ -204,7 +214,9 @@ export class TomcatController {
 
                 // tslint:disable-next-line:no-http-string
                 const serviceuri: string = `http://localhost:${serverPort}/${appName}`;
-                statusBar.text = `Open http://localhost:${serverPort}/${appName}`;
+                // tslint:disable-next-line:no-multiline-string
+                statusBar.text = `$(browser) Open in browser`;
```

Comment:
I think we can use '<string>' instead of `<string>` here, then ```// tslint:disable-next-line:no-multiline-string``` can be removed.

Reply:
Thanks! will try